.github/workflows/benchmark_visualization.yml

```diff
@@ -17,7 +17,7 @@ env:
 
 jobs:
   benchmark:
-    runs-on: ubuntu-20.04
+    runs-on: codebuild-soci-visualization-workflow-${{ github.run_id }}-${{ github.run_attempt }}-ubuntu-7.0-xlarge
```

Comment:
Jobs which run on CodeBuild runner would not be able to run in developers forks. It might be good to add a conditional for repository is `awslabs/soci-snapshotter`

Reply:
+1, looks like this isn't natively supported but there's workarounds to make this work

Reply:
This is a good call out, I have added a conditional statement to run the workflows on github runners if the trigger is on a fork.